opds_import.py

```diff
@@ -844,7 +849,8 @@ def consolidate_links(cls, links):
 
         Similarly if link n is a thumbnail and link n+1 is an image.
         """
-        new_links = list(links)
+        new_links = [x for x in links if x]
```

Comment:
This strips out any `None` elements (from `<link>` tags that were missing their `href`s).